src/icons/Plus.tsx

```diff
@@ -1,14 +1,23 @@
 import * as React from "react";
 
-export const Plus = () => (
+interface IconDefault {
+  color?: string;
+  scale?: number;
+}
+
+export const calculateDefaultDimensions = (x: number, scale = 100) => {
```

Comment:
**nitpick:** This method should be somewhere else - maybe create `helpers.ts` file in `icons` directory, and put it there